src/docs/admin/references/ServiceNowReference.adoc

```diff
@@ -0,0 +1,560 @@
+*Product Name:* ServiceNow +
+*Product Version:* Quebec +
+*Compatibility:* Tokyo
+
+==== Overview
+
+The ServiceNow integration provides means to establish a bidirectional communication
+between a ProActive server, and a ServiceNow instance.
+
+==== Objectives
+
+The goal is to provide templates and coding elements to help ServiceNow and ProActive engineers to easily integrate both solutions in their pipelines.
+
+It boils down to: +
+
+* Provide reusable code in ServiceNow to query ProActive's REST APIs.
+* Provide workflows in ProActive's catalog to query ServiceNow's REST APIs.
+
+==== Installation
+
+This section describes the available processes to set up the required files in ServiceNow in order to communicate
+with ProActive. The process choice is dictated by the architecture the integration will have in ServiceNow. Because ServiceNow
+is application-based, the natural course of action is to create a dedicated ProActive application.
+Otherwise, you might want to incorporate this integration in another application that already exists. In such case,
+you will have to manually create files in the host application, which means copy-paste code that will be presented in the next sections.
+
+===== Import from source control
+
+Working from an online repository is the most convenient and durable way to proceed.
+Importing an application from source control will result in a new application being created in this ServiceNow instance based on the remote repository you specify.
+The account credentials you supply must have read access to the remote repository
+
+. [.underline]#Lay down the Application#
+    .. Fork the https://bitbucket.org/activeeon/proactive-application[Application's repository]
+    .. Go to the Studio, under _System Applications_ -> _Studio_
++
+image::servicenow-studio-select.png[align="center"]
+    .. Click _Import from source control_
++
+image::servicenow-import-from-source-control.png[align="center"]
+    .. Click _Import_. When the application import is completed, click the _Select Application button_.
++
+image::servicenow-imported-from-source-control.png[align="center"]
+    .. In the Select Application dialog, click the link to the new application to open it for editing in Studio.
+
+Because you will very likely customize the application, it's a good idea to create a new branch an start working from there
```

Comment:
If you would like to customize the application, it is better to create a new branch and start working from there